test-suite/src/test/scala/org/scalajs/testsuite/jsinterop/ExportsTest.scala

```diff
@@ -1372,3 +1378,18 @@ object ExportObjSetterNamed_= { // scalastyle:ignore
   @JSExport
   val x = 1
 }
+
+@JSExport
```

Comment:
Why does this object need to be exported? If you don't export it, you can make it local to the `it` block. (Or am I missing some of the preconditions to actually trigger the scalac bug?)